spec/controllers/custom_bigbluebutton_rooms_controller_spec.rb

```diff
@@ -466,16 +466,6 @@
 
     context "if the user can record meetings in this room" do
       before(:each) { @ability.can :record_meeting, room }
-
```

Comment:
It's actually the other block that should be removed. The behavior should be as if the flag `webconf_auto_record` was always true, not false.